src/chatdlg.cpp

```diff
@@ -132,7 +132,7 @@ void CChatDlg::OnAnchorClicked ( const QUrl& Url )
     {
         if ( QMessageBox::question ( this,
                                      APP_NAME,
-                                     tr ( "Do you want to open the link %1 in an external browser?" ).arg ( " <b>" + Url.toString() + "</b> " ),
+                                     tr ( "Do you want to open the link '%1' in an external browser?" ).arg ( " <b>" + Url.toString() + "</b> " ),
```

Comment:
The question is if single quotes work…

Reply:
If you mean if single quotes can be used inside `tr()` with `%1`, yes it will work fine.

``` cpp
// output will be: "aaa 'bbb' ccc"
qDebug() << tr("aaa '%1' ccc").arg("bbb");
```

Reply:
Ok. Going to trust you on that ;-).

Reply:
```suggestion
                                     tr ( "Do you want to open the link '%1' in an external browser?" ).arg ( "<b>" + Url.toString() + "</b>" ),
```